controllers/runner_controller.go

```diff
@@ -237,6 +245,60 @@ func (r *RunnerReconciler) processRunnerCreation(ctx context.Context, runner v1a
 		return ctrl.Result{}, err
 	}
 
+	if r.ConfigureRunnersRBAC {
+		serviceAccount := &corev1.ServiceAccount{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      runner.ObjectMeta.Name + "-sa",
```

Comment:
```suggestion
				Name:      runner.ObjectMeta.Name,
```

I prefer not having a `-sa` prefix here, due to a reason similar to why I don't like naming a pod like `myrunner-pod` :)